Modules/_ctypes/stgdict.c

```diff
@@ -486,7 +486,7 @@ PyCStructUnionType_update_stgdict(PyObject *type, PyObject *fields, int isStruct
         size = offset = basedict->size;
         align = basedict->align;
         union_size = 0;
-        total_align = align ? align : forced_alignment;
+        total_align = max(align ? align : 1, forced_alignment);
```

Comment:
It may not work well in a macro.
```suggestion
        total_align = align ? align : 1;
        total_align = max(total_align, forced_alignment);
```

Reply:
Ah, I literally had this, but then thought it might be nicer to have it as a single line. I'll change it thanks.